spec/features/images_feature_spec.rb

```diff
@@ -0,0 +1,102 @@
+require 'rails_helper'
+
+feature 'Images' do
+
+  before do
+    sign_up_and_sign_in
+  end
+
+  context 'Website on initialization' do
+
+    scenario 'should display a prompt to add an image' do
+      visit '/images'
+      expect(page).to have_content 'No Images Yet'
+      expect(page).to have_link 'Upload Image'
+    end
+  end
+
+  context 'when image has been added in the database' do
+
+    before do
+      create_an_image
+    end
+
+    scenario 'will display image description' do
+      visit '/images'
+      expect(page).to have_content('My face')
+      expect(page).not_to have_content('No Images Yet')
+    end
+  end
+
+  context 'when user uploads images' do
+
+    scenario 'prompts them to enter a description, then displays description' do
+      create_an_image
+      expect(page).to have_content 'My face'
+      expect(current_path).to eq '/images'
+    end
+
+    scenario 'they must attach an image' do
+      visit '/'
+      click_link 'Upload Image'
+      fill_in 'Description', with: 'My face'
+      click_button 'Upload'
+      expect(page).not_to have_content 'My face'
+    end
+  end
+
+  context 'when user visits website' do
+
+    scenario 'lets them view uploaded images' do
+      create_an_image
+      sign_out
+      visit '/'
+      expect(page).to have_content 'My face'
+    end
+  end
+
+  context 'editing image descriptions' do
```

Comment:
Be consistent with context block formatting - you've used `when` so far, why stop now?